core/unsafe/console.js

```diff
@@ -33,16 +37,46 @@ colours.setTheme({
     title: ['green', 'bold']
 });
 
+var getTimestampString = function() {
+	var dt = new Date(),
+		diff = (dt.getTime() / 1000) - startupTime,
+		time = '[' + ('          ' + diff.toFixed(2)).slice(-10) + '] ';
```

Comment:
From what I gather this time stamp will measure seconds since startup, would it not be easier and better to record a datetimezone?